docs/synthetics/faq.md

```diff
@@ -5,13 +5,13 @@ description: FAQ about Sematext Synthetics, a synthetics monitoring solution for
 
 ### What should I do if I can't find the answer to my question in this FAQ?
 Check the [general FAQ](/faq) for questions that are not strictly
-about Sematext Synthetics.  If you can't find the answer to your
-question please email <support@sematext.com> or use our live chat.
+about Sematext Synthetics. If you can't find the answer to your
+question please email <support@sematext.com> or use our live chat in the bottom right corner of the page.
```

Comment:
On www yes, but not in SC, which is probably why we didn't try to be so specific.

Reply:
Yes, but here we're telling people to check the FAQ and contact support if their questions aren't answered there. FAQ also has the live chat down in the corner, which is why I figured that people are probably going to try and contact support there instead of going to SC. I can change it back to be less specific if you'd like, but that was the reasoning behind the change.

Reply:
Aha, OK, makes sense.